src/main/scala/reson/server/Server.scala

```diff
@@ -1,53 +1,64 @@
 package reson
 
-import com.twitter.finagle.exp.mysql.{Error, ServerError}
-import com.twitter.finagle.http.Method._
-import com.twitter.finagle.http._
-import com.twitter.finagle.http.Status._
-import com.twitter.finagle.http.path._
-import com.twitter.finagle.{SimpleFilter, Http, Service}
+import com.twitter.finagle.http.Method.{Get, Options}
+import com.twitter.finagle.http.Status.{Conflict, InternalServerError, NotFound}
+import com.twitter.finagle.http.path.{/, Path, Root}
+import com.twitter.finagle.http.{Request, Response}
+import com.twitter.finagle.mysql.ServerError
+import com.twitter.finagle.{Http, Service, SimpleFilter}
 import com.twitter.util.{Await, Future}
+import org.json4s.JsonAST.JValue
+import org.json4s.JsonDSL._
 import reson.db.MySQL
-import reson.server.{RequestParser}
-import util.Transformers._
-import rapture.json._
-import rapture.json.jsonBackends.jackson._
+import reson.server.RequestParser
+import reson.util.Transformers._
 
 /**
-  * Created by sscarduzio on 23/12/2015.
-  */
+ * Created by sscarduzio on 23/12/2015.
+ */
 object Server extends App {
-  def handle(req: Request, table: String, f: String => Future[String]): Future[Response] = {
+  def handle(req: Request, table: String,
+    f: String => Future[JValue]): Future[Response] = {
     for {
       query <- RequestParser.parse(req, table).toFuture
-      qString <- query.materialize.toFuture
-      dbResp <- f(qString)
+        qString <- query.materialize.toFuture
```

Comment:
what's the deal with these spaces?

Reply:
I'm sorry that was oversight on my part; IdeaJ reformatted the code and obviously, all rows inside for {} block should be aligned. Will fix and refresh pull request.